.github/workflows/android.yml

```diff
@@ -0,0 +1,95 @@
+name: Build test/ Android NDK r12b
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
It seems like `${{ github.event.pull_request.head.sha}}`  is being used in the rebuild required check steps, but not in pdebuild steps. 

https://github.com/nnstreamer/nnstreamer/blob/2a628fda7ad80edec0f7fa7fd6714643ab00bfea/.github/workflows/static.check.yml#L21

Shouldn't we use the same option in static check workflow?

Reply:
Thanks for the finding!